pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
I need to look but it might be that the plugin build template might send in false to this value via command line, so it might not have any effect on CI builds anyway.